docs/guides/js-waku/relay-send-receive.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Send and Receive Messages Using Relay
+---
+
+This guide provides detailed steps to set up a Waku node for sending and receiving messages using the [Relay protocol](/overview/concepts/protocols#relay).
+
+## Create a Waku Node
+
+Set up a Waku node by creating a relay node, connecting to network peers, choosing a [content topic](/overview/concepts/content-topics), and creating an `encoder` and `decoder` for [message encryption](https://rfc.vac.dev/spec/26/):
+
+```js
+import {
+	createRelayNode,
+	waitForRemotePeer,
+	createEncoder,
+	createDecoder
+} from "@waku/sdk";
+
+// Create and start a relay node
+const node = await createRelayNode({
+	defaultBootstrap: true, // bootstraps using pre-defined nodes
+	emitSelf: true, // emits sent message events to itself
+});
+await node.start();
+
+// Wait for a successful peer connection
+await waitForRemotePeer(node);
+
+// Choose a content topic
+const contentTopic = "/relay-guide/1/message/proto";
+
+// Create a message encoder and decoder
+const encoder = createEncoder({ contentTopic });
+const decoder = createDecoder(contentTopic);
+```
+
+:::info
+The `emitSelf` option emits sent message events to itself and invokes the node's subscribers.
+:::
+
+## Create a Message Structure
+
+Create your application's message structure using [Protobuf's valid message](https://github.com/protobufjs/protobuf.js#usage) fields:
+
+```js
+import protobuf from "protobufjs";
+
+// Create a message structure using Protobuf
+const ChatMessage = new protobuf.Type("ChatMessage")
+	.add(new protobuf.Field("timestamp", 1, "uint64"))
+	.add(new protobuf.Field("sender", 2, "string"))
+	.add(new protobuf.Field("message", 3, "string"));
+```
+
+:::info
+Please refer to the [Protobuf installation](/guides/js-waku/quick-start#create-a-message-structure) guide for steps on adding the `protobufjs` package to your project.
```

Comment:
Shall we add this protobufjs install guide in the home page along side `npm install @waku/sdk` as we want developers to use protobuf messages format?